features/support/env.rb

```diff
@@ -0,0 +1,22 @@
+require 'cucumber/rails'
+require "capybara-screenshot/cucumber"
+
+ActionController::Base.allow_rescue = false
+
+begin
```

Comment:
Do we need database cleaner? Doesn't look like it's required for any existing tests?